src/Practice2_Animal/Dog.java

```diff
@@ -0,0 +1,17 @@
+package Practice2_Animal;
+
+public class Dog extends Animal{
+
+	private String mauSac;
+	private int trongLuong;
+
+	public Dog(String ten, int tuoi, String giong, String mauSac, int trongLuong) {
+		super(ten, tuoi, giong);
+		this.mauSac = mauSac;
+		this.trongLuong = trongLuong;
+	}
+	public void Dog_info() {
+		System.out.println("Thông tin của chó gồm: " + "tên: " + getTen() + ", tuổi: " + getTuoi() + ", giống: " + getGiong() 
+		+ ", màu sắc: " + mauSac + ", trọng lượng: " + trongLuong);
+	}
```

Comment:
tương tự comment ở class Cat
mình cũng cần sửa lại thành như bên dưới nhé
```suggestion
        @Override
	public void displayInfo() {
		super.displayInfo();
		System.out.println(", màu sắc: " + mauSac + ", trọng lượng: " + trongLuong);
	}
```